FirebaseAuth/Tests/SampleSwift/SwiftApiTests/PhoneAuthTest.swift

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+//  PhoneAuthTest.swift
+//  SwiftApiTests
+
+import FirebaseAuth
+import Foundation
+import XCTest
+
+class PhoneAuthTests: TestsBase {
+  let kPhoneNumber = "+19999999999"
+  // This test verification code is specified for the given test phone number in the developer
+  // console.
+  let kVerificationCode = "777777"
```

Comment:
nit: the "k" is not preferred for Swift constants as opposed to Obj-C (see https://google.github.io/swift/#global-constants)